src/cli/commands/apps/index.ts

```diff
@@ -0,0 +1,94 @@
+import * as Debug from 'debug';
+import { MethodArgs } from '../../args';
+import { processCommandArgs } from '../process-command-args';
+import {
+  EValidSubCommands,
+  validAppsSubCommands,
+  createAppPrompts,
+  SNYK_APP_NAME,
+  SNYK_APP_REDIRECT_URIS,
+  SNYK_APP_SCOPES,
+  SNYK_APP_ORG_ID,
+  SNYK_APP_DEBUG,
+  ICreateAppOptions,
+  ICreateAppData,
+  AppsErrorMessages,
+} from '../../../lib/apps';
+import * as inquirer from '@snyk/inquirer';
+import { createApp } from './create-app';
+
+const debug = Debug(SNYK_APP_DEBUG);
+
+export default async function apps(
+  ...args0: MethodArgs
+): Promise<string | undefined | any> {
+  debug('Snyk apps CLI called');
+
+  const { options, paths } = processCommandArgs<ICreateAppOptions>(...args0);
+  debug(options, paths);
+
+  if (!options.experimental) throw new Error(AppsErrorMessages.useExperimental);
+
+  const commandVerb1 = paths[0];
+  const validCommandVerb =
+    commandVerb1 && validAppsSubCommands.includes(commandVerb1);
+
+  if (!validCommandVerb) {
+    // Display what is available
+    debug(`Not a valid sub command ${commandVerb1}`);
```

Comment:
I'd consider updating the error message a bit:

```suggestion
    debug(`Unknown subcommand: ${commandVerb1}`);
```